lib/Kernel.php

```diff
@@ -242,13 +242,12 @@ public function registerContainerConfiguration(LoaderInterface $loader)
             }
 
             foreach ($configArray as $config) {
-                $configKey = str_replace('-', '_', $config['defaultStorageDirectoryName']);
+                $configKey = $config['defaultStorageDirectoryName'];
                 if (!isset($containerConfig[self::CONFIG_LOCATION][$configKey])) {
                     continue;
                 }
-                $options = $containerConfig[self::CONFIG_LOCATION][$configKey]['write_target']['options'];
 
-                $configDir = rtrim($options['directory'] ?? self::getStorageDirectoryFromSymfonyConfig($containerConfig, $config['defaultStorageDirectoryName'], $config['storageDirectoryEnvVariableName']), '/\\');
```

Comment:
HI @robertSt7, does this mean that it is not possible to have a custom location anymore?

Reply:
Hi @lukmzig, no it was duplicate, because we get the directory from the symfony config also here https://github.com/pimcore/pimcore/pull/15291/files/c82012a07910cd02bc631d6d08fa902ae5b6a67c#diff-332ac2e54ccef891942249e9f2d654a5e8dc16de6f5243f11802991919d5396eL270

Reply:
Oh, I see now, thank you for the clarification!